src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sun calendar--month-length--28">
+      <div class="calendar__sell calendar__sell--1"></div>
```

Comment:
```suggestion
      <div class="calendar__cell"></div>
```
you don't need to add modifiers on cells, only on calendar

Reply:
Remove this modifier if you don`t need it now